parents.md

```diff
@@ -0,0 +1,50 @@
+# What is the WebDev Guild - A guide for parents
+
+Howdy, and welcome to the WebDev Guild! The WebDev Guild can be a great opportunity for your child to develop technical development skills that can benefit them now and in the future. Learning to code not only opens door for future career development, but can improve computational problem-solving skills in all areas. As your child sets out on this journey, we want you, as a parent, to be well-informed about the Guild, and your child’s learning.
+
+## What is the WebDev Guild?
+
+The WebDev Guild is a website programming club for kids. We provide resources and guides for students (we call them Guilders) to learn the basic principles of web development. The Guilders complete requirements and earn badges to showcase what they have accomplished. This isn't a watered-down program—the resources we provide are real-world guides created by industry leaders. Our goal is to develop Guilders who are able to assemble fully functional web apps by the time they complete all of the badges and requirements.
+
+Guilders are paired with experienced adult mentors who can answer their questions, guide their learning, and pass off their requirements.
```

Comment:
They are not necessarily paired with us.  More than anything, I would say that "They have access to experienced mentors" rather than they "are paired with"